docker/Dockerfile.geth

```diff
@@ -3,6 +3,7 @@ FROM ethereum/client-go:v1.12.0
 EXPOSE 8545
 
 COPY docker/gethData /
+COPY docker/deploymentOutput /
```

Comment:
these two lines seem only useful after you run `docker/scripts/deploy-docker.sh` first. but I don't see that in the Test Plan. so I am a bit confused how it works.

Reply:
`deploy_output.json` will be used in a later step - setup committee.